### DISCUSSION TITLE
How to set BackGround color for even and odd rows

### DISCUSSION THREAD

**Discussion by spn13** [mikekazakov/nimble-commander#488] [Q&A] [answered]
How to set color for even and odd rows for filePanelsListFocusedInactiveRowBackgroundColor from the settings interface in theme settings? When choosing a color, only one color is available. You can change it via json, but how to calculate the value for two colors?

**Comment by mikekazakov** [comment, marked as answer]
These colors are calculated as an alpha-based blend between the primary color and the background color (`filePanelsListRegularEvenRowBackgroundColor`/`filePanelsListRegularOddRowBackgroundColor`): https://github.com/mikekazakov/nimble-commander/blob/2d414fb6ef082c858a49936b9e8fe3680726de13/Source/NimbleCommander/NimbleCommander/States/FilePanels/List/PanelListViewRowView.mm#L127-L163

**Comment by spn13** [comment]
Thank you
